.gitignore

```diff
@@ -11,3 +11,4 @@ styleguide/*
 /types
 
 coverage
+/.idea/
```

Comment:
This is specific to your IDE. You can have a global `gitignore` to ignore your specific software. See [docs](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)